src/components/tab/styles/STab.tsx

```diff
@@ -0,0 +1,114 @@
+import styled, { css } from "styled-components";
+import {
+  CSSProgressiveBody02Semibold,
+  CSSProgressiveBody03Semibold,
+  CSSProgressiveCaption01Semibold,
+} from "../../../constants/styles/fonts";
+import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from "../../../constants/styles/mediaquerys";
+
+type ISTabSize = "Micro" | "Small" | "Medium" | "Default";
```

Comment:
@nicolasshiken I already saw this in different components. Please abstract this as a general type representing the sizes and make it accessible for all the application.

Reply:
Also make this lower case :pray:

Reply:
see src/components/button/Button.tsx